### PR TITLE
[6.6] Remove ECS aliases

### LINKED ISSUE #1794: Remove aliases in 6.6 after decision to go with a reindex strategy
We've decided to go with a reindexing strategy for upgrading to 7.0 and that means we need to clean up the aliases that were added for 6.6 which are no longer needed.

### PR DESCRIPTION
**Pull request by graphaelli**
reverts most of #1609, part of #1670, and all of #1665.

aliases that remain are inherited from libbeat processors:
```
# Common Beats fields
- name: event.timezone
  path: beat.timezone

# Cloud processor fields
- name: cloud.provider
  path: meta.cloud.provider
- name: cloud.instance.id
  path: meta.cloud.instance_id
- name: cloud.instance.name
  path: meta.cloud.instance_name
- name: cloud.machine.type
  path: meta.cloud.machine_type
- name: cloud.availability_zone
  path: meta.cloud.availability_zone
- name: cloud.project.id
  path: meta.cloud.project_id
- name: cloud.region
  path: meta.cloud.region

# Docker processor fields
- name: container.id
  path: docker.container.id
- name: container.name
  path: docker.container.name
- name: container.image.name
  path: docker.container.image
```

closes #1794 

will need a backport to 6.x

### CODE REVIEW

**Comment by simitt** [comment]
jenkins, retest this please

**Comment by graphaelli** [comment]
jenkins, retest this please

File: _beats/dev-tools/ecs-migration.yml [hunk 1]
```diff
@@ -15,6 +15,11 @@
 
 # Beat fields
 
+- from: beat.hostname
+  to: host.hostname
+  alias6: true
+  alias: true
```
Comment (simitt) [inline _beats/dev-tools/ecs-migration.yml]: Why are we aliasing this field and not including it in the reindexing script? And shouldn't `beat.hostname` go to `observer.hostname`?

Comment (graphaelli) [reply _beats/dev-tools/ecs-migration.yml]: Sorry, this is confusing.  This doesn't create an alias, it's just beats' log of their aliases.  The only aliases that are actually created are listed in the description.

**Pull request merged**